### ISSUE TITLE
Initiate calls via command line arguments?

### ISSUE THREAD

**Issue by j-dimension** [64characters/Telephone#642]
Hi,

is there an option to initiate a call with Telephone using command line arguments (with the target phone number being one argument)

Thanks,
Jens

**Comment by eofster** [comment]
Yes, it is. Telephone can be set as the handler for `tel:` URLs. This can be done by selecting it as "Standard for calls" in FaceTime preferences. When a `tel:` URL is being opened, the OS should activate Telephone and pass the URL to it. Here's how one can open a URL from Terminal:
```
open tel:1234567890
```
Note that `tel:` URLs don't have slashes.
